### DISCUSSION TITLE
Github Pages with Private Repo

### DISCUSSION THREAD

**Discussion by devjq** [community/community#47695] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi all, I have a quick question regarding using Github Pages with Private Repo. 
I understand that the regular free account users are not allowed. I am PRO user, but it still won't allowed me to do so. 
I wonder if there is any setting that I need to change or anything like that? Thank you in advance!

**Comment by yoannchaudet** [comment]
GitHub Pages is available in private repos for the Pro plan. If you can share a screenshot of your settings page, that may help diagnose the issue.

**Reply by devjq** [reply]
> GitHub Pages is available in private repos for the Pro plan. If you can share a screenshot of your settings page, that may help diagnose the issue.

i just tried again and it magically working with private repo now 👍
